src/sql/src/plan/statement/dml.rs

```diff
@@ -958,7 +958,8 @@ pub fn describe_copy(
     CopyStatement { relation, .. }: CopyStatement<Aug>,
 ) -> Result<StatementDesc, PlanError> {
     Ok(match relation {
-        CopyRelation::Table { name, columns } => describe_table(scx, name, columns)?,
+        // This isn't necessarily a table - it must be a catalog item type that produces data.
+        CopyRelation::Table { name, columns } => describe_item_for_copy(scx, name, columns)?,
```

Comment:
Doesn't this want to fail if called on a non-table for `COPY TO`?

Reply:
Ah yes my bad - you're right